### DISCUSSION TITLE
Creating new issues on github directly adds the good first issue label

### DISCUSSION THREAD

**Discussion by sidrao2006** [firebase/flutterfire#5632] [Feature request]
Creating new issues using github issue templates automatically adds the good first issue label. This label is supposed to be added by curators and people with Contributor access.To denote that this is a user's first issue, a new label named - user's first issue, or simply - first issue,  can be created

**Comment by darshankawar** [comment]
Hi @sidrao2006,
The `good first issue` label is applied for Doc related issues and not all issues. If newbie developers want to start contributing to the project, this label helps them to get a start. 
Regarding `Needs Attention` label, this one helps the curators to filter, prioritize and respond as needed on all new issues logged.
Hope this helps to answer your question.
Thanks.

**Comment by sidrao2006** [comment]
@darshankawar adding the labels is not a problem. But the names are misleading.

To identify bug, there is a label.
To identify that the issue was recently created, a `new issue` label can be created and added automatically through both the templates.
The `good first issue` should be added by the curators if its a new contributor who's first issue contributes to the repository.
A `first issue` label can be added through the template if it's a contributor's first issue
